components/org.wso2.carbon.appmgt.api/src/main/java/org/wso2/carbon/appmgt/api/APIProvider.java

```diff
@@ -53,23 +52,50 @@ public void deleteBusinessOwner(String businessOwnerId) throws
 
      /* *
      *Get business ower data.
-     * @param ownerId
+     * @param businessOwnerId
      * @return
      * @throws AppManagementException
      */
-    public Map<String, String> getBusinessOwnerCustomProperties(int ownerId) throws AppManagementException;
+    public List<BusinessOwnerProperty> getBusinessOwnerCustomProperties(int businessOwnerId) throws
+                                                                                             AppManagementException;
     /**
      * @return
      * @throws AppManagementException
      */
     public List<BusinessOwner> getBusinessOwners() throws AppManagementException;
 
     /**
+     * Return the owner properties of the given owner Id.
```

Comment:
Java doc comments issues and formatting issues are fixed in https://github.com/wso2/carbon-appmgt/pull/539/files